guide/src/templates/hermes-version.md

```diff
@@ -1 +1 @@
-v1.6.0+1c1cf02
+v1.7.0
```

Comment:
Could we start appending the commit hash to the version, as per #3614?

Reply:
The thing is we don't know it before we merge the release PR, and since we do the release from master we can't guess it.

I don't see what it brings to users anyway, it's mostly for us when we cut special builds. I would actually suggest we remove it from release builds altogether, as it gives the impression that it's not a final build.

What do you think?